components/gitpod-db/src/typeorm/migration/1657722262390-RenameWorkspaceIdColumn.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class RenameWorkspaceIdColumn1657722262390 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`DROP INDEX \`IDX_e759ab5fcf57350da51fcf56bc\` ON \`d_b_workspace_instance_usage\``);
+        await queryRunner.query(`DROP INDEX \`IDX_25d77dfa246b93672c317e26ad\` ON \`d_b_workspace_instance_usage\``);
+        await queryRunner.query(`DROP INDEX \`IDX_1358af969a29fd9e0c6cabf37c\` ON \`d_b_workspace_instance_usage\``);
+        await queryRunner.query(`DROP TABLE \`d_b_workspace_instance_usage\``);
```

Comment:
:bulb: I think you can also do a 3-way-move with just dropping the PRIMARY KEY constraint (but maintaining the invariant that a table has to have at least one all the time). Sharing in case it's helpful in future scenarios, but this works here as well. :+1:

Reply:
You can specify those in `[db-workspace-instance-usage.ts](https://github.com/gitpod-io/gitpod/pull/11353/files#diff-30691deb4aa982d2d0f5b5aa25e12296a1a308aa7a2681bbe92232199bfa945b)` `@Index(name)`, and they will get generated in here.